src/agcOnReady.js

```diff
@@ -3,6 +3,7 @@
     angular.module('googlechart')
         .directive('agcOnReady', onReadyDirective);
         
+    onReadyDirective.$inject = ['$timeout'];
     function onReadyDirective(){
```

Comment:
Looks like you're registering a dependency here, but didn't include a method argument to receive it. This should be throwing an error for `$timeout` is not a function when the callback actually runs.
```suggestion
    function onReadyDirective($timeout){
```